index.html

```diff
@@ -270,15 +274,13 @@ <h3>
                         Skills
                     </h3>
                 </div>
-                <div class="topic-list">
-                    <ul class="list-about">
-                        <li class="list-tag">Codekit</li>
-                        <li class="list-tag">GitHub</li>
-                        <li class="list-tag">Codepen</li>
-                        <li class="list-tag">Gitlab</li>
-                        <li class="list-tag">Terminal</li>
-                    </ul>
-                </div>
+                <ul class="list-about">
+                    <li class="list-tag">Codekit</li>
+                    <li class="list-tag">GitHub</li>
+                    <li class="list-tag">Codepen</li>
+                    <li class="list-tag">Gitlab</li>
+                    <li class="list-tag">Terminal</li>
+                </ul>
             </div>
         </div>
         <footer class="contact-footer"> 
```

Comment:
Kindly fix the contact section's style to match the Figma design too 🤗

Your design:

![image](https://user-images.githubusercontent.com/73714615/187662165-a6a6526b-3c02-488a-8216-247e554fe23c.png)

Figma design:

![image](https://user-images.githubusercontent.com/73714615/187662214-74cb4b73-e72b-4c8f-97e4-eb51db314d5e.png)